aws_lambda_powertools/utilities/feature_flags/time_conditions.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime, timezone
+from typing import Dict, List
+
+from .schema import TimeValues
+
+HOUR_MIN_SEPARATOR = ":"
+
+
+def _get_utc_time_now() -> datetime:
+    return datetime.now(timezone.utc)
+
+
+def compare_between_utc_days(action: str, values: List[str]) -> bool:
+    current_day = _get_utc_time_now().strftime("%A").upper()
+    return current_day in values
+
+
+def compare_utc_datetime_range(action: str, values: Dict) -> bool:
+    current_time_utc: datetime = _get_utc_time_now()
+    start_date = datetime.strptime(values.get(TimeValues.START.value, ""), "%Y-%m-%dT%H:%M:%S%z")
+    end_date = datetime.strptime(values.get(TimeValues.END.value, ""), "%Y-%m-%dT%H:%M:%S%z")
```

Comment:
We already use this datetime format string in multiple places, including in the `ConditionsValidator` class. Can we move it to a more global constant and use it everywhere, instead of repeating it?

Reply:
yeah i refactored it and didnt use it here. np